fix(details): typo in component css variable (#3405)
digdir/designsystemet: .changeset/moody-bugs-do.md

---
"@digdir/designsystemet-css": patch
---

> fixed the typo in `--dsc-details-border-block-wdith`. The old variable is kept for now, so there are no breaking changes

Reviewer: ```suggestion
**Details**: Added `--dsc-details-border-block-width` to fix typo in `--dsc-details-border-block-wdith`.
The old variable is kept as is, so there are no breaking changes.
```
Not a fan of my own wording here, but lets at least show the new variable as well

Author: Yes better 👍
